index.html

```diff
@@ -1151,6 +1155,40 @@ <h2>Constrainable Properties</h2>
         </tbody>
       </table>
     </section>
+    <section>
+      <h2>Capture timestamps</h2>
+      <div>
+          Some video sources can supply information about when a video frame was captured.
+          This information is useful for example for AV sync and end-to-end delay measurement.
```

Comment:
How does it help with AV sync if it's only on video frames?

Reply:
Absolute audio capture timestamps are available via other web APIs right?
To try to answer the overarching question, presentation timestamps alone are insufficient if you want to do accurate AV sync on a local (think MediaRecorder) or on a remote receiver, especially in the presence of delays in the capture process, or insert VTGs in the capture pipeline to do video processing.
Is this answer enough or do you want to see further changes to clarify this in the spec text based on latest uploaded revision?